src/esp/sensor/PinholeCamera.cpp

```diff
@@ -28,8 +29,37 @@ void PinholeCamera::setProjectionParameters(SensorSpec::ptr spec) {
   hfov_ = std::atof(spec_->parameters.at("hfov").c_str());
 }
 
-void PinholeCamera::setProjectionMatrix(gfx::RenderCamera& targetCamera) {
+PinholeCamera& PinholeCamera::setProjectionMatrix(
+    gfx::RenderCamera& targetCamera) {
   targetCamera.setProjectionMatrix(width_, height_, near_, far_, hfov_);
+  return *this;
+}
+
+PinholeCamera& PinholeCamera::setModelViewMatrix(
```

Comment:
It makes sense. Will change the function names to make them consistent.

Thanks!

Reply:
Agreed on the naming.  I also think this isn't the modelview matrix?   i.e. the modelview matrix transformations points from model coordinates to camera coordinates, but I believe the matrix here transforms from camera coordinates to model coordiantes.

Reply:
> modelview = view x model;
> Here, the view is an Identity (because the "camera" (sensor) orientation and position is already considered in the model matrix).
> Here, the transformation matrix (model matrix) is to transfer any vertices from the model local frame to the world frame.

@erikwijmans: Actually what I said in the previous message was wrong. I apologize.
Here, we set the transformation matrix (denoted as T) from sensor to the camera. So T is the transformation matrix for the "render camera" now. T will transform from camera frame to the world frame ( you are right at this point). 
The actual view matrix = inverse(T), which will be computed and applied by the magnum library.

Reply:
Yeah, this is what I was originally thinking and wonder if Aman had run into this, but apparently webVR calls the modelview matrix the matrix that transforms from camera coordinates into the world frame, so who knows if the convention I am use to is actually universal or not.